typescript/t.d.ts

```diff
@@ -128,12 +128,13 @@ export type ParseKeys<
   KPrefix = undefined,
   Keys extends $Dictionary = KeysByTOptions<TOpt>,
   ActualNS extends Namespace = NsByTOptions<Ns, TOpt>,
+  const Context extends TOpt['context'] = TOpt['context'],
```

Comment:
The added `const` here throws an [error TS1277: 'const' modifier can only appear on a type parameter of a function, method or class](https://github.com/maevsi/stomper/actions/runs/8605657170/job/23582395255?pr=203#step:7:304)

Reply:
I'm also encountering into the same problem.

Reply:
This issue will be tracked via #2168